#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 0bd3de7eac926caf00cd090b1770ed157dd27659 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ingo=20Kl=C3=B6cker?= <dev@ingo-kloecker.de>
Date: Tue, 28 Jan 2025 15:06:37 +0100
Subject: [PATCH] Make gpgme++ include future-proof

The gpgme++ prefix will soon be mandatory for GpgME++ includes.
---
 client/editor/composerviewbase.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/editor/composerviewbase.cpp b/client/editor/composerviewbase.cpp
index 3fb5edc..02ea765 100644
--- a/client/editor/composerviewbase.cpp
+++ b/client/editor/composerviewbase.cpp
@@ -43,7 +43,7 @@
 #include <QGpgME/ExportJob>
 #include <QGpgME/ImportJob>
 #include <QGpgME/Protocol>
-#include <global.h>
+#include <gpgme++/global.h>
 #include <gpgme++/context.h>
 #include <gpgme++/importresult.h>

--
2.48.1
